livekit-agents/livekit/agents/voice/agent_session.py

```diff
@@ -625,18 +625,18 @@ async def drain(self) -> None:
 
         await self._activity.drain()
 
-    def _close_soon(
+    def shutdown(
```

Comment:
I think we should create a separate method, so we don't expose the `drain` and the `reason` to users